test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/io/ReadersTest.scala

```diff
@@ -364,3 +364,4 @@ class InputStreamReaderTest {
     assertThrows(classOf[IOException], r.mark(0))
   }
 }
+
```

Comment:
Spurious change here, but otherwise LGTM.

Please squash all the commits in a single one with a good commit message.

Reply:
Should be ok now.